docs/quick_start.md

```diff
@@ -99,22 +96,30 @@ In this case, we consider the [Hello World agent service](./hello_world_agent_se
         ]
         ```
 
-2. Use the CLI to download and build the images to deploy the [Hello World agent service](./hello_world_agent_service.md):
+
+3. Run a local deployment for the [Hello World agent service](./hello_world_agent_service.md):
```

Comment:
Please, split this step into 2 steps and follow the convention used in the rest of the docs:
first, detail exactly what the command is going to do.
second, present the command.

Also it's confusing that this step is called "run a local deployment", and in step 4 we have "run the deployment".